# Raw Data from WG035 F/T input via WG006 (gripper MCB).
int32[] data          # Array of 6 raw values from ADC.  Values are currenty signed 16bit values.  
                      # In future values may be averaged/filtered in firmware   
int64 sample_count    # Counts number of samples  